src/User32/User32+DISPLAY_DEVICE.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright © .NET Foundation and Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="DISPLAY_DEVICE"/> nested type.
+    /// </content>
+    public partial class User32
+    {
+        /// <summary>
+        /// Receives information about the display device specified by the <c>iDevNum</c> parameter of the <see cref="User32.EnumDisplayDevices"/> function.
+        /// </summary>
+        [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Unicode)]
+        public struct DISPLAY_DEVICE
+        {
+            /// <summary>
+            /// Size, in bytes, of the DISPLAY_DEVICE structure. This must be initialized prior to calling <see cref="User32.EnumDisplayDevices"/>.
+            /// </summary>
+            public uint cb;
+
+            /// <summary>
+            /// An array of characters identifying the device name. This is either the adapter device or the monitor device.
+            /// </summary>
+            [MarshalAs(UnmanagedType.ByValTStr, SizeConst = 32)]
+            public string DeviceName;
```

Comment:
We don't use `[MarshalAs]` attributes in structs. See our [contribution guidelines](https://github.com/dotnet/pinvoke/blob/1bcc7b190e8dd66c7daa43334adb457a0322b2e5/CONTRIBUTING.md#structs).